pallets/dapp-staking-v3/src/lib.rs

```diff
@@ -211,6 +211,14 @@ pub mod pallet {
         #[pallet::constant]
         type RankingEnabled: Get<bool>;
 
+        /// Minimum acceptable percent of total issuance for dynamic tier thresholds.
+        #[pallet::constant]
+        type MinTotalIssuancePercent: Get<Perbill>;
+
+        /// Maximum acceptable percent of total issuance for dynamic tier thresholds.
+        #[pallet::constant]
+        type MaxTotalIssuancePercent: Get<Perbill>;
```

Comment:
I've read the comment that this is extra safety check, but I'd suggest to omit it.
This isn't critical to the working of the pallet - worst case the threshold goes too low and some dApps get higher rewards than expected.

It's good to be extra safe in critical parts (e.g. as we are with the inflation), but here it seems like an overkill to me. I'd suggest to remove it.